docs/firststeps/SSH-keys.md

```diff
@@ -0,0 +1,133 @@
+    (**Tools :octicons-arrow-right-16: MobaKeyGen**). Both tools are identical.
+    
+    In order to generate your key pairs for LUMI, choose the option RSA and
+    set the number of bits to 4096. The, press the *Generate* button.
+
+    <figure>
+      <img src="../../assets/images/win-keygen-step1.png" width="400" alt="Create SSH key pair with windows - step 1">
+    </figure>
+
+    You will be requested to move the mouse in the Key area to generate some 
+    entropy; do so until the green bar is completely filled.
+
+    <figure>
+      <img src="../../assets/images/win-keygen-step2.png" width="400" alt="Create SSH key pair with windows - step 2">
+    </figure>
+
+    After that, enter a comment in the Key comment field and a strong
+    passphrase. Please choose a secure passphrase. It should be at least 8
+    (preferably 12) characters long and should contain numbers, letters and
+    special characters. **Do not leave the passphrase empty**.
+
+    <figure>
+      <img src="../../assets/images/win-keygen-step3.png" width="400" alt="Create SSH key pair with windows - step 3">
+    </figure>
+
+    The next step is to save your public and private key. Click on the *Save 
+    public key* button and save it to the desired location (for example, with 
+    `id_rsa_lumi.pub` as a name). Do the same with your private key by clicking
+    on the *Save private key* button and save it to the desired location (for 
+    example, with `id_rsa_lumi` as a name).
+
+!!! warning
+    The private key should never be shared with anyone, not even
+    with LUMI staff. It should also be stored only on your local computer
+    (public key can be safely stored in cloud services). Protect it with a good
+    password! Otherwise, anyone with access to the file system can steal your
+    SSH key.
+
+## Register your public key
+
+=== "For regular users"
+
+    Now that you have generated your key pair, you need to register your **public** key
+    in your [:material-account: **MyAccessID user profile**][myaccessid-profile]. From there, the public key will be 
+    copied to LUMI.
+
+    To register your key, click on the *Settings* item of the menu on the left
+    as shown in the figure below. Then select *SSH keys* and click the *New key* button. Now copy and paste the content of your **public** key file in the text area and click the *Add SSH key* button.
+
+    <figure>
+      <img src="../../assets/images/MyAccessID_ssh-key.png" width="480" alt="Screenshot of user profile settings to setup ssh public key">
+      <figcaption>MyAccessID Own profile information to add ssh public key.</figcaption>
+    </figure>
+
+=== "For users with a Finnish allocation"
+
+    Now that you have generated your key pair, you need register your 
+    **public** key in your MyCSC [:material-account: **user profile**][mycsc-profile].
+
+    To register your key with [MyCSC][mycsc-profile], click on *My Profile* item
```

Comment:
minot typo? missing 'the' before *My Profile*?

Reply:
Fixed in [86d205b](https://github.com/Lumi-supercomputer/lumi-userguide/pull/51/commits/86d205b806a38105172989fb412cf2142c27420a).